modules/common/src/main/java/org/opencastproject/serviceregistry/api/Incidents.java

```diff
@@ -160,14 +161,14 @@ public void recordFailure(Job job, int code, Throwable t, List<Tuple<String, Str
    */
   public void recordFailure(Job job, int code, Throwable t, Map<String, String> params,
           List<Tuple<String, String>> details) {
-    List<Tuple<String, String>> detailList = Mutables.list(details);
+    List<Tuple<String, String>> detailList = new LinkedList<>(details);
```

Comment:
I don't really care per se, but the "default" List implementation for Java seems to be `ArrayList` rather than `LinkedList`, probably because of better memory alignment and thus caching performance (see `Collectors.toList()`, which returns an `ArrayList`, for example, just as `toSet()` returns a `HashSet` instead of a `TreeSet`). But again, no blocker for me.